lib/chef/knife/ec2_server_create.rb

```diff
@@ -1054,6 +1065,9 @@ def create_server_def
           server_def[:block_device_mapping] = (server_def[:block_device_mapping] || []) << {'VirtualName' => "ephemeral#{i}", 'DeviceName' => device_name}
         end
 
+        ## Termination Protection cannot be enabled for spot instances ##
+        server_def[:disable_api_termination] = locate_config_value(:disable_api_termination) if locate_config_value(:spot_price).nil?
```

Comment:
Okay. Might add clarity to set the comment to something like "Cannot pass disable_api_termination to API when using spot instances".